js/views/modals/purchase/Payment.js

```diff
@@ -84,8 +84,8 @@ export default class extends BaseVw {
     return {
       'click .js-payFromWallet': 'clickPayFromWallet',
       'click .js-payFromAlt': 'clickPayFromAlt',
-      'click .js-copyAmount': 'copyAmount',
-      'click .js-copyAddress': 'copyAddress',
+      'click .js-amountRow': 'copyAmount',
+      'click .js-addressRow': 'copyAddress',
```

Comment:
The click action was changed to the containing element so all parts of it are clickable.